### PR TITLE
GITC-73:  remove unused imports

### PR DESCRIPTION
**Pull request by chibie**
##### Description

Brief reason why we should remove unused imports

https://medium.datadriveninvestor.com/java-why-you-remove-unused-imports-7a3327a0b877

This was done using a combination of reports from Vulture and Pylance (VSCode extension)

This is the vulture report → https://rentry.co/uumip

#### some extra care taken

1. hacks for circular imports aren’t regarded as unused
2. modular imports in apps.py or other places that aren’t auto imported aren’t regarded as unused
3. i discovered other shit codes but intentionally limited this pr to the specific purpose

example use of pylance -> https://www.loom.com/share/35bc02a1c6b04e249cb17fc3517551d1

### CODE REVIEW

File: app/dashboard/tests/test_binance_sync.py [hunk 1]
```diff
@@ -67,7 +64,7 @@ def setUp(self):
         )
         BountyFulfillment.objects.create(
             fulfiller_address='0x0000000000000000000000000000000000000000',
-            bounty=bounty,
+            bounty=self.bounty,
```
Comment (thelostone-mc) [inline app/dashboard/tests/test_binance_sync.py:67]: was this intentional ?

File: app/marketing/apps.py [hunk 2]
```diff
@@ -26,4 +26,4 @@ class MarketingConfig(AppConfig):
     name = 'marketing'
 
     def ready(self):
-        from .signals import create_email_subscriber
+        import .signals # noqa
```
Comment (thelostone-mc) [inline app/marketing/apps.py:29]: was this intentional ?

File: app/kudos/test_models.py [hunk 3]
```diff
@@ -1,22 +0,0 @@
-# -*- coding: utf-8 -*-
-"""Test the Kudos models.
```
Comment (zls2) [inline app/kudos/test_models.py:2]: @chibie Good catch that this test isn't implemented. Should we make a ticket to implement this test?

Comment (zacheryschiller) [inline app/kudos/test_models.py:2]: @chibie Good catch that this test isn't implemented. Should we make a ticket to implement this test?

File: app/dashboard/tests/test_binance_sync.py (hunk 1 above)
Comment (chibie) [reply app/dashboard/tests/test_binance_sync.py:67]: yes

File: app/kudos/test_models.py (hunk 3 above)
Comment (chibie) [reply app/kudos/test_models.py:2]: @zacheryschiller i don't think it's worth the effort to test just models... more focus should be on testing request/response cycle and helper functions

**Review by thelostone-mc — APPROVED** [review]

**Review by zacheryschiller — APPROVED** [review]

**Pull request closed without merging**